### PR TITLE
expand axis order discussion with examples

### PR DESCRIPTION
**Pull request by jodygarnett**
This is currently an addition to the docs expanding the WFS axis order discussion with examples, highlighting combinations that produce unexpected results.

### CODE REVIEW

**Comment by jodygarnett** [comment]
The above failures:
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.22.2:test (default-test) on project gs-wms: There are test failures.
```

Should be unrelated to this documentation change.

File: doc/en/user/source/services/wfs/axis_order.rst [hunk 1]
```diff
@@ -5,38 +5,56 @@ Axis ordering
 
 WFS 1.0.0 servers return geographic coordinates in longitude/latitude (x/y) order, the most common way to distribute data. For example, many shapefiles adopt this order by default. 
 
-However, the traditional axis order for geographic and cartographic systems is the opposite—latitude/longitude (y/x)—and the later WFS specifications respect this. The default axis ordering support is:
+However, the traditional axis order for geographic and cartographic systems is the opposite—latitude/longitude (y/x) — and the later WFS specifications respect this. The default axis ordering support is:
```
Comment (bradh) [inline doc/en/user/source/services/wfs/axis_order.rst]: I think something like:

However, the traditional axis order for geographic and cartographic systems is the opposite: latitude/longitude (y/x),  and the later WFS specifications respect this

Comment (jodygarnett) [reply doc/en/user/source/services/wfs/axis_order.rst]: Thanks I will resolve it to have a similar structure as the proceeding sentence:

> However, the traditional axis order for geographic and cartographic systems in latitude/longitude (y/x) order, and the later WFS specifications respect this.

File: doc/en/user/source/services/wfs/axis_order.rst [hunk 2]
```diff
@@ -49,7 +67,33 @@ The default data directory includes the following dataset illustrating this chal
...
+  
+   The :file:`PRJ` did not provide an axis-order and GeoSever has filled in an assumption. This describing the data in `x/y` order which matches our data and we could use it unmodified.
+   
+   .. figure:: img/native_srs.png
+      
+      Native SRS for topp:states
+
+#. Locate :guilabel:`Declared SRS` and click on :guilabel:`EPSG:WGS 84...` link to see the definition used to publish this content.
    
+   This is the internal definition of ``EPSG:4326`` as shown in the SRSList above, which also describes the data in `x/y` order matching our data. This definition provides slightly more readable names along with additional  ``AUTHORY`` information that may be helpful to client applications.
```
[the remark this answers is not in the document]
Comment (jodygarnett) [reply doc/en/user/source/services/wfs/axis_order.rst]: fixed

**Comment by jodygarnett** [comment]
Updated, thanks for the feedback @bradh, rebased in the hopes of getting a passing build.

**Comment by jodygarnett** [comment]
Build still fails on both AppVeyor and Travis on the same three tests...

File: doc/en/user/source/services/wfs/axis_order.rst [hunk 3]
```diff
@@ -0,0 +1,676 @@
+.. _wfs_basics_axis:
+
+Axis ordering
+=============
+
+WFS 1.0.0 servers return geographic coordinates in longitude/latitude (x/y) order, the most common way to distribute data. For example, many shapefiles adopt this order by default.
```
Comment (aaime) [inline doc/en/user/source/services/wfs/axis_order.rst]: Simply talking about longitude and latitude is limiting and gives false impressions. There is a number of projected CRSs that are north/east oriented in their EPSG definition, those are affected by the axis order rules too. Would be better to talk about "respecting EPSG axis order" vs "forcing east/north" order. It is also to be noted that there are CRSs, like the polar stereographic based ones, where talking about east and north makes no sense, having a pole in the middle of the CRS axis.

**Comment by jodygarnett** [comment]
Andrea fixed build upstream at geotools level (failed build resulted in conflicting copies of jai-ext being pulled in).

**Pull request merged**